index.ts

```diff
@@ -303,6 +376,35 @@ async function listNeurons() {
   }
 }
 
+async function mergeNeurons(sourceNeuronId: bigint, targetNeuronId: bigint) {
+  const identity = await getLedgerIdentity();
+  await assertLedgerVersion({ identity, minVersion: "2.1.7" });
+  const governance = GovernanceCanister.create({
+    agent: await getAgent(identity),
+    hardwareWallet: false,
+  });
+
+  await governance.mergeNeurons({
+    targetNeuronId: BigInt(targetNeuronId),
```

Comment:
out of curiousity, `targetNeuronId` and `sourceNeuronId` are not already `bigint` so you cast them to be double sure?

Reply:
I believe that initially there were numbers and I forgot to refactor.

Reply:
thanks!